webstart.ts

```diff
@@ -1,96 +1,66 @@
-import {BasicREPL} from './repl';
+import { BasicREPL } from './repl';
 import { Type, Value } from './ast';
 import { defaultTypeEnv } from './type-check';
 import { NUM, BOOL, NONE } from './utils';
+// import CodeMirror from "codemirror";
+import { renderResult, renderError, renderPrint } from "./ouputrender";
 
-function stringify(typ: Type, arg: any) : string {
-  switch(typ.tag) {
-    case "number":
-      return (arg as number).toString();
-    case "bool":
-      return (arg as boolean)? "True" : "False";
-    case "none":
-      return "None";
-    case "class":
-      return typ.name;
-  }
-}
 
-function print(typ: Type, arg : number) : any {
-  console.log("Logging from WASM: ", arg);
-  const elt = document.createElement("pre");
-  document.getElementById("output").appendChild(elt);
-  elt.innerText = stringify(typ, arg);
-  return arg;
-}
+console.log('this is normoal')
+import CodeMirror from "codemirror";
+import "codemirror/addon/edit/closebrackets";
+import "codemirror/mode/python/python";
+import "codemirror/addon/hint/show-hint";
+import "codemirror/addon/fold/foldcode";
+import "codemirror/addon/fold/foldgutter";
+import "codemirror/addon/fold/brace-fold";
+import "codemirror/addon/fold/comment-fold";
+import "./style.scss";
 
 function assert_not_none(arg: any) : any {
   if (arg === 0)
     throw new Error("RUNTIME ERROR: cannot perform operation on none");
   return arg;
 }
 
+console.log('this is ok')
```

Comment:
Clean up on all console.log or other debugging statements

Reply:
Done, thank you!